assets/js/components/Layout/Layout.jsx

```diff
@@ -89,13 +90,19 @@ const Layout = () => {
                 onClick={handleSidebar}
                 title={isCollapsed ? 'Expand Sidebar' : 'Collapse Sidebar'}
               >
-                <EOS_KEYBOARD_DOUBLE_ARROW_LEFT
-                  size="24"
-                  color="currentColor"
-                  className={classNames('text-gray-400 hover:text-gray-300', {
-                    '-scale-100': isCollapsed,
-                  })}
-                />
+                {isCollapsed ? (
```

Comment:
@dottorblaster So I am now using two implementations of the `EOS Keyboard Double Arrow` icons but it @arbulu89 had a suggestion to create a React component for both icons so that we can reduce the repeated attributes for size, color and classname. What do you think?

Reply:
@jagabomb a component would be an overkill to me, maybe we can try just defining these things as `const`?

Reply:
@dottorblaster Cool, I have defined the three const values in a new commit.